proposals/discriminated-unions.md

```diff
@@ -157,21 +157,20 @@ on repeated invocations.
 
 ### Switch expression and patterns
 
-There are some proposed adjustments to pattern matching and the switch expression to handle
+There are some proposed adjustments to pattern matching and the `switch` expression to handle
 `enum classes`. Switch expressions can already match types through the variable pattern, but
-for currently for reference types, no set of switch arms in the switch expression are considered
+for reference types, currently no set of switch arms in the switch expression are considered
 complete, except for matching against the static type of the argument, or a subtype.
 
-Switch expressions would be changed such that, if the root type of an `enum class` is the static
+Switch expressions would be changed such that if the root type of an `enum class` is the static
 type of the argument to the switch expression, and there is a set of patterns matching all
 members of the enum, then the switch will be considered exhaustive.
 
 Since value members are not constants and do not define new static types, they currently cannot
 be matched by pattern. To make this possible, a new pattern using the constant pattern syntax
-will be added to allow match against `enum class` value members. The match is defined to succeed
+will be added to allow matching against `enum class` value members. A match is defined to succeed
 if and only if the argument to the pattern match and the value returned by the `enum class` value
-member would be reference equal, although the implementation is not required to perform this
-check.
+member are reference equal, although the implementation is not required to perform this check.
```

Comment:
Please revert this bit.

Reply:
I think it's better if I just withdraw this, and you do what you want.

Reply:
If you want to. There are still some fixes left we'd be happy to take.